shell/src/main/java/org/jtaccuino/jshell/extensions/dependency/Dependencies.java

```diff
@@ -68,9 +68,12 @@ static List<Path> resolve(String mavenCoordinates) {
             Dependency dependency
                     = new Dependency(new DefaultArtifact(mavenCoordinates), JavaScopes.COMPILE);
             RemoteRepository central = new RemoteRepository.Builder("central", "default", "https://repo1.maven.org/maven2/").build();
+           RemoteRepository mavenlocal = new RemoteRepository.Builder("mavenLocal","default", "file:" + System.getProperty("user.home")
```

Comment:
Just curious but shouldn't the URI start with `file:///`
Why not use `File.toURI().toString()` or `Path.toUri().toString()` and have the JDK handle the formatting of the URIString?